crates/oxc_allocator/src/arena.rs

```diff
@@ -14,7 +14,7 @@ use serde::{ser::SerializeSeq, Serialize, Serializer};
 
 use crate::Allocator;
 
-/// Bumpalo Box
+/// A Box without `Drop`
```

Comment:
Need to update to the following:

A Box without `Drop`. This is used for over coming self-reference structs.

It is a memory leak if the boxed value has a `Drop` implementation.